include/deal.II/grid/grid_tools.h

```diff
@@ -425,9 +425,23 @@ namespace GridTools
    * hanging nodes in ghost cells if you call the current functions: The
    * vertices of all locally owned cells will be correct, but the vertices of
    * some ghost cells may not. This means that computations like
-   * KellyErrorEstimator may give wrong answers. A safe approach is to use
-   * this function prior to any refinement in parallel, if that is possible,
-   * but not after you refine the mesh.
+   * KellyErrorEstimator may give wrong answers.
+   *
+   * @note This function is in general not compatible with manifolds attached
+   * to the triangulation. If you want to perform refinements according to a
+   * manifold description, you should first do the refinements, subsequently
+   * deactivate all manifolds, and finally call the transform() function. The
```

Comment:
I don't think this statement on refinements is true in this generality - it is perfectly valid to refine the mesh after the transformation if one has a manifold for the *transformed* geometry at hand. So I would state this as
```suggestion
   * to the triangulation. If you want to perform refinements according to a
   * manifold description, you either need to attach the appropriate manifold
   * for the transformed geometry and clear possibly old manifolds, or
   * do the refinements before the transformation, deactivate all manifolds,
   * and finally call the transform() function. The
```

Reply:
I can incorporate this idea. However, I would reformulate it a bit because the sentence becomes quite complicated now.